Source/Core/Core/ConfigManager.h

```diff
@@ -51,7 +51,7 @@ struct BootParameters;
 #define BACKEND_NULLSOUND _trans("No Audio Output")
 #define BACKEND_ALSA "ALSA"
 #define BACKEND_CUBEB "Cubeb"
-#define BACKEND_OPENAL "OpenAL"
+#define BACKEND_OPENAL "OpenAL (Deprecated)"
```

Comment:
```suggestion
#define BACKEND_OPENAL _trans("OpenAL (Deprecated)")
```

The word "deprecated" should be translated.

Reply:
This is actually "wrong" because the enabled backend will then be saved with the translated name, and revert back to default if you change language. What should be done instead is, use that string in the inside code, but just translate it when you show it in the UI.
The is a QT combo that adds the list of all these backend names, so it should be translated there on the spot, if it is possible. Something like backends_combo->AddItem(QTranslateString(backend_names_iterator));

Reply:
> What should be done instead is, use that string in the inside code, but just translate it when you show it in the UI.

That is exactly how it would work if you used my suggestion. The `_trans` macro does nothing at runtime (it is used only because it tells gettext to put this string in the list of translatable strings), and DolphinQt uses `tr` to actually translate the string.

Reply:
ah ok. I thought it would create some kind of static function that based on some global language setting would return a different literal string built into the source code.